docs: added alpha jetbrains gateway coder plugin | remove code-server version parameter (#5900)
coder/coder: docs/ides/web-ides.md

    os             = "linux"
    startup_script = <<EOF
    #!/bin/sh
    # install code-server
    # add '-s -- --version x.x.x' to install a specific code-server version
    curl -fsSL https://code-server.dev/install.sh | sh

>     # start code-server on a specific port
>     # authn is off since the user already authn-ed into the coder deployment

Reviewer: ```suggestion
    # start code-server on a specific port
    # auth is disabled since the user already authenticated via Coder
```
